http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieRuntimeMonitor.scala

```diff
@@ -48,37 +54,70 @@ class ZombieRuntimeMonitor[F[_]: Parallel: ContextShift: Timer](
       startInstant = Instant.ofEpochMilli(start)
       semaphore <- Semaphore[F](config.concurrency)
 
-      // Get active clusters from the Leo DB, grouped by project
-      clusterMap <- ZombieMonitorQueries.listZombieQuery.transaction
+      // Get active runtimes from the Leo DB, grouped by project
+      activeRuntimeMap <- ZombieMonitorQueries.listActiveZombieQuery.transaction
 
       _ <- logger.info(
-        s"Starting zombie detection across ${clusterMap.size} projects with concurrency of ${config.concurrency}"
+        s"Starting active zombie detection across ${activeRuntimeMap.size} projects with concurrency of ${config.concurrency}"
       )
-      zombies <- clusterMap.toList.parFlatTraverse[F, PotentialZombieRuntime] {
-        case (project, clusters) =>
+      activeZombies <- activeRuntimeMap.toList.parFlatTraverse[F, ZombieCandidate] {
+        case (project, zombieCandidates) =>
           semaphore.withPermit(
             // Check if the project is active
             isProjectActiveInGoogle(project).flatMap {
               case true =>
-                // If the project is active, check each individual cluster
-                clusters.toList.traverseFilter { cluster =>
-                  isRuntimeActiveInGoogle(cluster, startInstant).ifA(F.pure(None), F.pure(Some(cluster)))
+                // If the project is active, check each individual runtime
+                zombieCandidates.toList.traverseFilter { candidate =>
+                  // If the runtime is less than one minute old, it may not exist in Google because it is still Provisioning, so it's not a zombie
+                  isRuntimeActiveInGoogle(candidate.googleProject, candidate.runtimeName, candidate.cloudService, startInstant)
+                    .ifA[Option[ZombieCandidate]](F.pure(None), F.pure(zombieIfOlderThanOneMinute(candidate, startInstant)))
```

Comment:
How did we decide on 1 minute? Seems a little aggressive -- I could imaging a 1 minute lag between when front Leo receives the create request and back Leo actually creates the runtime. Maybe 10 minutes would be safer.